docs/components/connectors/out-of-the-box-connectors/rabbitmq.md

```diff
@@ -0,0 +1,103 @@
+
+:::note
+Ensure you write out the correct exchange name and routing key, because the **RabbitMQ Connector** can't throw an exception if they are incorrect.
+:::
+
+## Create a RabbitMQ Connector task
+
+To use the **RabbitMQ Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide to using Connectors](../use-connectors.md) to learn more.
+
+## Make your RabbitMQ Connector for sending messages executable
+
+To make the **RabbitMQ Connector** executable, choose the needed connection type in the **Authentication** section and fill out the mandatory fields highlighted in red in the properties panel:
+
+![connectors-rabbitmq-red-properties](../img/connectors-rabbitmq-red-properties.png)
+
+:::note
+All the mandatory and non-mandatory fields depending on the authentication selection you choose are covered in the upcoming sections.
+:::
+
+## Authentication
+
+You can choose among the available RabbitMQ Connectors according to your authentication requirements.
+First, you must have a user in your RabbitMQ instance with the necessary permissions. See more at the [RabbitMQ access control specification](https://www.rabbitmq.com/access-control.html).
+
+Next, we will choose a type of connection.
+
+### URI type connection
+
+For a URI connection, take the following steps:
+
+1. Click the **URI** connection type in the **Authentication** section
+2. Set **URI** to `URI`. It must contain RabbitMQ username, password, host name, port number, and virtual host. For example, `amqp://userName:password@serverHost:port/virtualHost`; follow the [RabbitMQ URI specification](https://www.rabbitmq.com/uri-spec.html) to learn more.
+
+![connectors-rabbitmq-uri-fields](../img/connectors-rabbitmq-uri-fields.png)
+
+### Credentials type connection
+
+For a credentials connection, take the following steps:
+
+1. Click the **Username/Password** connection type in the **Authentication** section
+2. Set the **Password** to `Password`.
+   ![connectors-rabbitmq-credentials-fields](../img/connectors-rabbitmq-credentials-fields.png)
+
+## Routing data
+
+In the **Routing** section you must set routing data (free feel use JSON format and FEEL expression format):
+
+```json
+{
+  "exchange": "<your-rabbitmq-exchange-name>",
+  "routingKey": "<your-rabbitmq-routing-key-name>",
+  "virtualHost": "<your-rabbitmq-virtual-host-name>",
+  "hostName": "<your-rabbitmq-host-name-or-ip-address>",
+  "port": "<your-rabbitmq-port-number>"
+}
+```
+
+For a **URI** type connection, the required fields are `exchange` and `routingKey`.
+For a **Credentials** type connection, all fields are required.
+
+![connectors-rabbitmq-routing](../img/connectors-rabbitmq-routing.png)
+
+## Message
+
+1. Set your message payload to **Message** section. Message can be text and JSON format.
```

Comment:
What do you mean here? Do you mean, "In the **Message** section, insert the message payload."?

Reply:
Added in my latest commit.